src/quo2/components/buttons/dynamic_button.cljs

```diff
@@ -93,8 +92,8 @@
                                                 :notification-up   0
                                                 :search-with-label 8)}}
            (case type
-             :jump-to                                       (i18n/label :t/jump-to)
-             :search-with-label                             (i18n/label :t/back)
+             :jump-to                                       label
+             :search-with-label                             label
```

Comment:
Please can you elaborate, why this change required.

Reply:
good question, i wanted quo library as pure as possible, so it shouldn't depend on status translation, so we need to pass labels as parameters to component, but there might be difficulties for sure, but at least we should try, probably there is better implementation for dynamic button, but my goal was just to move i18n outside in this PR

Reply:
cool, sounds good. Thank you